### DISCUSSION TITLE
Triage Base permission for Github Organizations

### DISCUSSION THREAD

**Discussion by feanil** [community/community#57863] [Repositories]
### Select Topic Area

Product Feedback

### Body

I love the fact that you can set a base permission for all members of a GitHub organization.  However, I was surprised to see that you can't set the default base permission to "Triage"
![image](https://github.com/community/community/assets/781561/85d7fcee-eacf-4224-a2ac-319eff74d8af)

Being able to have a default base permission of triage would make it a lot easier to ensure that any one can start helping to triage issues even if they don't have a lot of coding experience and shouldn't be allowed to edit the code.

**Comment by toddjonker** [comment]
I have found this quite frustrating as I launch a new org.  At present, ensuring all members have triage permissions requires fiddly toil from the admin for each new person, hampered by having no bulk-change to member roles.

**Comment by gaardhus** [comment]
even ChatGPT has internalized the horrible UX of organization management in GitHub.

<img width="534" height="83" alt="image" src="https://github.com/user-attachments/assets/3d54c51f-e53a-4e24-82b9-08f6b5a9e0f1" />
